### PR TITLE
Added doxygen script for audio framework

### PR DESCRIPTION
**Pull request by JeonginKim**
Added doxygen script for audio framework

### CODE REVIEW

File: framework/include/tinyalsa/limits.h [hunk 1]
```diff
@@ -43,6 +43,18 @@
...
+/**
+* @defgroup TinyAlsa TinyAlsa
+* @ingroup TinyAlsa
+* @brief Provides APIs for Audio Framework
+* @{
+ */
+
+/**
+ * @file limits.h
+ * @brief All macros, structures and functions that make up the PCM interface.
```
Comment (sunghan-chang) [inline framework/include/tinyalsa/limits.h]: All of brief are same. Do they have same fucntionality? Then why do we split files?

File: framework/include/tinyalsa/limits.h [hunk 2]
```diff
@@ -43,6 +43,18 @@
 ** DAMAGE.
 */
 
+/**
+* @defgroup TinyAlsa TinyAlsa
```
Comment (jeongchanKim) [inline framework/include/tinyalsa/limits.h]: you already defined the TinyAlsa group.
you'd better remove this defgroup

File: framework/include/tinyalsa/interval.h [hunk 3]
```diff
@@ -43,6 +43,18 @@
 ** DAMAGE.
 */
 
+/**
+* @defgroup TinyAlsa TinyAlsa
+* @ingroup TinyAlsa
+* @brief Provides APIs for Audio Framework
+* @{
```
Comment (jeongchanKim) [inline framework/include/tinyalsa/interval.h]: you opened the '{', but there is no '}'

File: framework/include/tinyalsa/limits.h [hunk 4]
```diff
@@ -43,6 +43,18 @@
...
+* @ingroup TinyAlsa
+* @brief Provides APIs for Audio Framework
+* @{
```
Comment (jeongchanKim) [inline framework/include/tinyalsa/limits.h]: you opened '{', but not closed.

**Comment by Taejun-Kwon** [comment]
@shivgarg Currently we don't need it, I guess we'd better remove them.
Please consider removing or need to fix doxygen style.

**Comment by shivgarg** [comment]
Ok, I will remove limits.h and interval.h file.

**Pull request merged**